srcpkgs/olm-python3/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'olm-python3'
 pkgname=olm-python3
-version=3.2.8
-revision=2
+version=3.2.13
+revision=1
 build_wrksrc=python
 build_style=python3-module
```

Comment:
```suggestion
build_style=python3-module
make_check_args="--deselect tests/group_session_test.py::TestClass::test_encrypt
 --deselect tests/group_session_test.py::TestClass::test_decrypt"
```

Reply:
this is definitely better than what we had before, but tbh, these are two really important tests for this library. They test core functionality, and deselecting those doesn't feel quite right.

Reply:
Looking at some repos, to include those tests you need the pytest-benchmark and then package [aspectlib](https://pypi.org/project/fields/) which requires packaging [fields](https://pypi.org/project/fields/). After that, theoretically all the tests should pass :laughing:

Reply:
Or you could apply this patch which runs the tests without benchmarking thus removing the need of any extra dependency. Apply at your own risk!
```patch
--- a/python/tests/group_session_test.py
+++ b/python/tests/group_session_test.py
@@ -62,14 +62,12 @@
         index = inbound.first_known_index
         assert isinstance(index, int)
 
-    def test_encrypt(self, benchmark):
-        benchmark.weave(OutboundGroupSession.encrypt, lazy=True)
+    def test_encrypt(self):
         outbound = OutboundGroupSession()
         inbound = InboundGroupSession(outbound.session_key)
         assert "Test", 0 == inbound.decrypt(outbound.encrypt("Test"))
 
-    def test_decrypt(self, benchmark):
-        benchmark.weave(InboundGroupSession.decrypt, lazy=True)
+    def test_decrypt(self):
         outbound = OutboundGroupSession()
         inbound = InboundGroupSession(outbound.session_key)
         assert "Test", 0 == inbound.decrypt(outbound.encrypt("Test"))
```

Reply:
I could, but I'd expect that the function passed to benchmark.weave is going to have some sort of side effect. And at this point, we're putting in *way* more effort into this than just packaging pytest-benchmark. What I've not mentioned though, is that for this specific case to work we also need aspectlib, which needs fields, both of which aren't packaged yet. I think I'll package these later on though, so that these test can just work as they are right now.

Reply:
I've added aspectlib and fields now, these tests now work as they should.